app/views/users/_language_settings.html.erb

```diff
@@ -8,8 +8,8 @@
   <div class="checkbox-field">
     <% Languages::LIST.each do |code, name| %>
       <div class="sub-field">
-        <%= check_box_tag "user[preferred_languages][]", code, @user.preferred_languages_array.include?(code) %>
-        <%= label_tag "user[preferred_languages][]", name %>
+        <%= check_box_tag "preferred_languages_#{code}", code, @user.preferred_languages_array.include?(code) %>
```

Comment:
Hmmm I believe this will affect the input value/

Reply:
oh no. I just checked this out here and, yeap, it crashed. Later on I'll read the errors with more time and try to figure this out 😅

Thanks for reviewing ❤️